src/main/java/seedu/sellsavvy/logic/commands/personcommands/EditPersonCommand.java

```diff
@@ -94,7 +100,15 @@ public CommandResult execute(Model model) throws CommandException {
             model.updateSelectedPerson(editedPerson);
         }
 
-        return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedPerson)));
+        String feedbackToUser = model.hasSimilarPerson(editedPerson)
```

Comment:
Should this also include `editPersonDescriptor.isNameEdited()`?
The check for similar tags included that condition.

Reply:
in